locale/es_ES/reader.xml

```diff
@@ -40,18 +40,37 @@
 	<message key="rt.admin.contexts.createContext">Crear contexto</message>
 	<message key="rt.admin.contexts.edit.editContext">Editar contexto</message>
 	<message key="rt.admin.contexts.metadata">Metadatos</message>
-	<message key="rt.admin.contexts.options.authorTerms">Usar nombres de autores como términos de búsqueda por defecto para permitir a los usuarios encontrar otros trabajos de los autores de este artículo (p.ej., para un contexto de "Otros trabajos")</message>
+	<message key="rt.admin.contexts.options.authorTerms">Usar nombres de autores como términos de búsqueda por defecto para permitir a los usuarios encontrar otros trabajos de los autores de este artículo (p.ej., para un contexto de &quot;Otros trabajos&quot;)</message>
```

Comment:
Likewise, I'm nervous here about using the `quot` entity where `"` is perfectly fine. In some cases it'll probably cause the `&quot;` to be displayed verbatim in the output and/or emails where it's included.